app/src/main/res/layout/activity_login.xml

```diff
@@ -1,20 +1,60 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context="mcs.salazar.jesus.mcsbattleship.LoginActivity">
 
+    tools:context="mcs.salazar.jesus.mcsbattleship.LoginActivity"
+    android:orientation="horizontal">
 
-    <com.facebook.login.widget.LoginButton
-        android:id="@+id/login_facebook_button"
-        android:layout_width="wrap_content"
```

Comment:
It looks like you are deleting facebook button... do we want this to happen? I need another dev to run in a different environment... Ivan, can you run this branch and make sure your login to facebook is still working?

Reply:
No, I just move the facebook button to the bottom of email and password widget